ui/cap-react/src/antd/forms/widgets/DateWidget.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+
+import { DatePicker } from "antd";
+
+import moment from "moment";
+
+const DATE_FORMAT = "DD/MM/YYYY";
+const DATE_TIME_FORMAT = "DD/MM/YYYY HH:mm:ss";
+
+const DateWidget = ({
+  autofocus,
+  disabled,
+  formContext,
+  id,
+  onBlur,
+  onChange,
+  onFocus,
+  readonly,
+  value,
+  schema,
+}) => {
+  const { readonlyAsDisabled = true } = formContext;
+
+  const [defaultFormat, setDefaultFormat] = useState();
+
+  useEffect(
+    () => {
+      setDefaultFormat(schema.allowTime ? DATE_TIME_FORMAT : DATE_FORMAT);
```

Comment:
maybe the date-time should be taken from the `format` (e.g. `date`, `date-time` - https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times)